### DISCUSSION TITLE
How to set maxDiskUsage on Broker

### DISCUSSION THREAD

**Discussion by eanbiso** [arkmq-org/arkmq-org-broker-operator#831] [Q&A] [answered]
Hi support,
I would need to understand how I can configure the "maxDiskUsage" property on the Broker, for being able to change its default value (90).
Since I saw the property (maxDiskUsage) is managed by the [ConfigurationImpl.java](https://github.com/apache/activemq-artemis/blob/main/artemis-server/src/main/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImpl.java) class, 
I though it was possible to tune this value via the broker properties in a way like this:
```
Name:         amq-broker
Namespace:    edge
API Version:  broker.amq.io/v1beta1
Kind:         ActiveMQArtemis
Spec:
  Broker Properties:
    name=amq-broker
    maxDiskUsage=95
```
but, when I checked the broker.xml file on the pods (e.g. looking at the file amq-broker-ss-0:amq-broker/etc/broker.xml), I still found the parameter set to its default value (90)
```
<max-disk-usage>90</max-disk-usage>
```
Am I doing something wrong?
I would like to understand the right way to configure properties like this, since we would be interested to tune also other ones like "minDiskFree" and "globalMaxSize".

Many thanks in advance for your help

**Comment by brusdev** [comment, marked as answer]
Hi @eanbiso, your example is quite correct and it is the only way to set the property `maxDiskUsage`. The broker properties doesn't affect the broker xml, you can double-check the `maxDiskUsage` property value querying the jolokia api, i.e.

```
apiVersion: broker.amq.io/v1beta1
kind: ActiveMQArtemis
metadata:
  name: amq-broker
spec:
  brokerProperties:
    - maxDiskUsage=95
```

```
curl -u admin:admin 'http://amq-broker-ss-0:8161/console/jolokia/read/org.apache.activemq.artemis:broker="amq-broker"/MaxDiskUsage'
```

**Reply by eanbiso** [reply]
Thanks!
